src/app/components/SubjectHeading/SubjectHeadingsTableBody.jsx

```diff
@@ -94,7 +92,11 @@ class SubjectHeadingsTableBody extends React.Component {
         updateParent: () => this.updateRange(range, interval, 'end', 10),
       });
     }
-    return subjectHeadingsInInterval;
+    return (
+      <React.Fragment>
+        { subjectHeadingsInInterval.map(item => this.tableRow(item)) }
+      </React.Fragment>
+    );
```

Comment:
What kind of behavior prompted this solution?

Reply:
- I noticed that sometimes when sorting, the button appeared outside the list of subject headings, which made me wonder if it was being re-rendered by the table body component or just sticking around
- I added a data field set to a random number which allowed me to see that it wasn't being re-rendered
- I checked the errors and noticed the warning about keys, at which point I put the two things together
- Taking a look at the list elements, I noticed that there were two down arrows
- Since there is only ever one up or down arrow per interval, I realized the key issue could be fixed by scoping each arrow to a particular interval
- Since we don't want to introduce a new html element for the interval, React.Fragment made sense as a solution

Reply:
I am currently working on `development` for the business continuity work. I want to play around with this a bit, but will have to do it next week.